app.rb

```diff
@@ -1,3 +1,4 @@
+# -*- encoding : utf-8 -*-
```

Comment:
:+1: It's also necessary because forcing utf-8 actually removes Spanish characters we use like ó